src/tracing.c

```diff
@@ -145,9 +147,12 @@ static mrb_value mrb_seccomp_start_ptrace(mrb_state *mrb, mrb_value self) {
                       "Something is wrong when grandchildren fork");
           }
         } else if (MRB_PTRACE_EVENT(status) == PTRACE_EVENT_SECCOMP) {
-          if (mrb_seccomp_on_tracer_trap(mrb, hook, child) < 0) {
+          if (mrb_seccomp_on_tracer_trap(mrb, hook, child, detach) < 0) {
             mrb_raise(mrb, E_RUNTIME_ERROR, "Something is wrong in trap event");
           }
+	  if (detach) {
+            return self;
+	  }
```

Comment:
This indent includes `"\t"` . Would you use `clang-format` to fix?

Reply:
OK! I apply clang-format and fix it.